tailwind/tailwind.css

```
@@ -546,6 +546,14 @@
   }
 }
 
+/*--- map loading marker ------------------------- */
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)